server/core/services.py

```
@@ -0,0 +1,87 @@
+        Log out a user and destroy the session
+        
+        Args:
+            request: The HTTP request object
+            
+        Returns:
+            dict: User data including authentication token if used
+        """
+        try:
+            # Log the user out (validates with HTTP session storage)
+            logout(request)
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)